platforms/intellij/compose/playground/src/jvmTest/kotlin/foundry/intellij/compose/playground/MarkdownPlaygroundKtTest.kt

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (C) 2024 Slack Technologies, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package foundry.intellij.compose.playground
+
+import androidx.compose.ui.test.ExperimentalTestApi
+import androidx.compose.ui.test.onNodeWithTag
+import androidx.compose.ui.test.onRoot
+import androidx.compose.ui.test.performClick
+import androidx.compose.ui.test.runDesktopComposeUiTest
+import com.github.takahirom.roborazzi.RoborazziOptions
+import io.github.takahirom.roborazzi.captureRoboImage
+import kotlinx.coroutines.test.runTest
+import org.junit.Test
+
+class MarkdownPlaygroundKtTest {
```

Comment:
Here is the test, based on the sample one: https://github.com/takahirom/roborazzi?tab=readme-ov-file#experimental-feature-compose-desktop-support

Reply:
I used Intellij's "generate test" feature so it would place it in the right directory. It auto-generated the name. I can rename it to just `MarkdownPlaygroundTest`. <img width="632" alt="Screenshot 2024-10-28 at 12 08 59 PM" src="https://github.com/user-attachments/assets/e460baee-9584-4733-a38a-01ecdfa3f8d1">